daemon/logger/logger.go

```diff
@@ -15,4 +21,6 @@ type Logger interface {
 	Log(*Message) error
 	Name() string
 	Close() error
+	Read() (io.Reader, error)
+	LogPath() string
```

Comment:
@LK4D4 I agree, too. however apparently Remote API `GET /container/xxx` has a field like `"LogPath": "..."` in the response json because I guess nobody thought we could have drivers that don't log to a path, in the past and added that to the API. Now we can't remove that field b/c back-compat...

This is the cleanest solution I found.